_posts/0000-01-02-tarnyates.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!'
+---
+Your text
```

Comment:
"I can think of no more uplifting fact than the ability of a man to improve his life by a conscious effort"